package.json

```diff
@@ -142,10 +141,10 @@
     "ts-node": "^2.0.0",
     "tslint": "~4.3.1",
     "typedoc": "^0.5.3",
-    "typescript": "2.0.10",
+    "typescript": "~2.1.5",
```

Comment:
Unless Angular has changed their stance on this and resolved the AOT issues,  this should not be changed as last I checked it isn't officially supported until Angular 4.x